### PR TITLE
frescobaldi 2.18.1

### PR DESCRIPTION
**Pull request by DomT4**
Finishes up #39107.

Didn't realise `Lilypond` was still recommended in the core, I thought this had been merged already - Apologies.

### CODE REVIEW

**Comment by DomT4** [comment]
CC @mikemcquaid re https://github.com/Homebrew/homebrew/pull/39107#issuecomment-97507101.

File: Library/Formula/frescobaldi.rb [hunk 1]
```diff
@@ -34,17 +38,21 @@ def install
...
+  def caveats
+    s = <<-EOS.undent
+      By default, a splash screen is shown on startup; this causes the main
+      window not to show until the application icon on the dock is clicked
+      (Cmd-Tab application switching does not appear to work). You may
+      want to disable the splash screen in the preferences to solve this
+      issue. See:
+        https://github.com/wbsoft/frescobaldi/issues/428
+    EOS
+    return s
```
Comment (mikemcquaid) [inline Library/Formula/frescobaldi.rb]: Don't need the `return` or the `s` declaration; in Ruby the last value is implicitly returned from any function.

**Comment by mikemcquaid** [comment]
Other than the nit :+1:

File: Library/Formula/frescobaldi.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/frescobaldi.rb]: Heh, my brain skipped over the caveats entirely :smile_cat:, Will fix.

**Comment by DomT4** [comment]
Should be good now. Will let the bot trundle back over it, and merge it sometime after.

**Pull request closed without merging**